app/models/category.rb

```diff
@@ -1,3 +1,7 @@
 class Category < ActiveRecord::Base
-	has_many :videos, -> {order :title}
```

Comment:
Ah, the same happened in this file. I figured out that you have not converted some of your files to 2 spaces instead of tabs. Make sure you do :-)